README.md

```diff
@@ -2,43 +2,84 @@
 
 [![build status][travis-image]][travis-url]
 
-This modules allows you to check if a number is a valid.
+This modules provides some tools to handle `Social Security Numbers` from the `United States of America`.
 
 ## Installation
 
 Choose your preferred method:
 
-* npm: `npm install --save is-valid-ssn`
-* Download: [is-valid-ssn](https://github.com/seegno/is-valid-ssn)
+* npm: `npm install --save ssn-validator`
+* Download: [ssn-validator](https://github.com/seegno/ssn-validator)
 
-## Usage
+## Methods
 
-*NOTE:* The input number **must not** be formatted to `xxx-xxxxxx`.
+### `isValid(value, [options])`
 
-> Check if number is valid.
+This method validates if the given value is a valid `Social Security Number`.
 
+#### Arguments
+
+1. `value` *(&#42;)*: The value to check.
+2. `[options]` *(Object)*: The options object.
+3. `[options.strict=true]` _(boolean)_: Whether or not to reject formatted numbers.
```

Comment:
`Whether or not formatting characters such as dashes or spaces should be accepted.`